### ISSUE TITLE
Feature request: Add postpone break to tray menu

### ISSUE THREAD

**Issue by arthurmcgregor** [tom-james-watson/breaktimer-app#13]
I have my postpone break time set to 1hr which is useful to hide notifications during meetings/video calls. I realise I can trigger break postponement via the notification, but often I miss the notification so I would like another way to postpone the break for 1hr. I can use the tray menu to disable breaks entirely, but I would like to be able to postpone breaks (for my custom duration) at any time, rather than having to wait for a notification (which I might miss). The downside to simply disabling breaks is that I usually forget to re-enable them when I return to my desk!

PS Great app - thanks!

**Comment by tom-james-watson** [comment]
Yeah this is a good idea. I think some setting next to `Disable` like `Temporarily disable...` which opens a sub menu with `For 1 hour`, `Till tomorrow` for example.

**Comment by AugustasV** [comment]
Yes, that would be great! Agree.

**Comment by NavDiv** [comment]
Last Christmas, I didn't have Breaktimer & strained my shoulder badly (for months in pain) while working relentlessly on a project & forgetting to take breaks. Since then, Breaktimer has saved me many times. However, without this function, after a live gig, I often forget to re-enable for sometimes a week. Therefore I nearly made the same mistake as last Christmas again :( This function is very crucial to me... Cheers

**Comment by AugustasV** [comment]
@NavDiv probably it will not be implemented. Recommend using Stretchly instead.

**Comment by tom-james-watson** [comment]
A PR is very welcome

**Comment by gousto01** [comment]
This option would really make the app perfect. I forget to turn it on too many times after I disable it for a meeting or watching a movie.

**Comment by sandrb** [comment]
@AugustasV why won't it be implemented? It would mean the world to me! Happy to code it myself actually!

**Comment by AugustasV** [comment]
@sandrb , it's just my guess. If you can code, feel free to do so. A pull request is very welcome!

**Comment by sandrb** [comment]
@AugustasV You have any guarantee that such a pull request will actually be merged into master and make it to production? Don't want to waste my time on a feature that won't actualy be implemented.

**Comment by tom-james-watson** [comment]
Maintainer here - yep, well implemented PR would definitely be merged and published!

**Comment by tom-james-watson** [comment]
It only took 5 years, but this was finally implemented and has been released as part of version 1.3.0 🎉
